tasks/push-to-cdn/push-to-cdn.yaml

```diff
@@ -0,0 +1,110 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: push-to-cdn
+  labels:
+    app.kubernetes.io/version: "4.1.0"
```

Comment:
Any reason to start with `4.1.0`? We typically start with `0.1.0`.

Reply:
cp/ps error from collect-data, updated, ty!